bpf/process/bpf_execve_event.c

```diff
@@ -269,6 +269,26 @@ execve_rate(void *ctx)
 	return 0;
 }
 
+// update bitset mark
+FUNC_INLINE
+void update_mb_bitset(struct execve_map_value *curr)
+{
+	__u64 *bitsetp;
+	struct execve_map_value *parent;
+
+	parent = event_find_parent();
+	if (parent) {
+		// initialize bitset to parent's
+		curr->bin.mb_bitset = parent->bin.mb_bitset;
+	}
```

Comment:
Having a comment about why you do that instead of what is done could make this easier to get. If I understand here you do the case where the parent (let's say `/bin/bash`) was `In` MatchBinaries with `followChildren`. So you want the children to bear the bitset mark so that we can trigger an event on them, so you pass the bitset to the child.

Reply:
Good point, added:
```
+               /* ->mb_bitset is used to track matchBinary matches to children (followChildren), so
+                * here we propagate the parent values to the child.
+                */

```